docker/build_and_test/Dockerfile.FEDORA

```diff
@@ -0,0 +1,35 @@
+# Recipe for an HE-ready Fedora 33 docker
+FROM fedora:33
+
+# Install general HElib pre-requisites
+RUN yum -y upgrade && \
+    yum -y update && \
+    yum -y group install "Development Tools" && \
+    yum -y install g++ m4 patchelf cmake3 wget python3 diffutils parallel
+
+# Install bats-core (used for testing utils and examples)
+RUN cd && \
+    wget https://github.com/bats-core/bats-core/archive/v1.2.1.tar.gz && \
+    tar xf v1.2.1.tar.gz && \
```

Comment:
on fedora and ubuntu bats-core you can install with yum and apt-get.  Not sure if you did this for consistency, but maybe its best to keep it consistent.

Reply:
That's a good point. The reason I do it this way is because there are two versions of bats
https://github.com/bats-core/bats-core
and
https://github.com/sstephenson/bats
The latter which I think is not being maintained anymore and I believe is the one that apt-get and (maybe?) yum install.
This method removes any possible ambiguity.